projects/ff4j-properties/src/app/properties.module.ts

```diff
@@ -0,0 +1,27 @@
+import {Injector, NgModule} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { PropertiesComponent } from './properties.component';
+import { environment } from '../environments/environment';
+import {createCustomElement} from '@angular/elements';
```

Comment:
Please follow a proper spacing and indentation

Reply:
👍